docs/contributor/07-module-migration-guide.md

```diff
@@ -1,11 +1,18 @@
 # New Module Submission and Promotion Process: Migration Guide
 
-To ensure a smooth transition, the submission pipeline, and Lifecycle Manager support **both** the old and new metadata formats. If both are present, Lifecycle Manager prefers the new format. If no new metadata format is provided, it falls back to the old channel-based metadata.
+To ensure a smooth transition, the submission pipeline, Lifecycle Manager (KLM), and Busola support **both** the old and new metadata formats. If both are present, KLM and Busola use the new format. If no new metadata format is provided, it falls back to the old channel-based metadata.
```

Comment:
In case it's not 3 elements but 2, consider this:

```suggestion
To ensure a smooth transition, the submission pipeline - Lifecycle Manager (KLM), and Busola - support **both** the old and new metadata formats. If both are present, KLM and Busola use the new format. If no new metadata format is provided, it falls back to the old channel-based metadata.
```